xdsl/dialects/scf.py

```diff
@@ -118,6 +119,49 @@ def get(
         return op
 
 
+@irdl_op_definition
+class ParallelOp(Operation):
+    name = "scf.parallel"
+    lowerBound: Annotated[VarOperand, IndexType]
+    upperBound: Annotated[VarOperand, IndexType]
+    step: Annotated[VarOperand, IndexType]
+    initVals: Annotated[VarOperand, AnyAttr()]
+    results: Annotated[VarOpResult, AnyAttr()]
+
+    body: Region
+
+    irdl_options = [AttrSizedOperandSegments()]
+
+    @staticmethod
+    def from_bounds_and_body(lowerBounds: Sequence[SSAValue | Operation],
```

Comment:
Can't this be just `get(` ?

Reply:
It could, but I'm really not a fan of this name myself. Do you prefer it?